packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -185,7 +191,7 @@ class Auth {
         // Special endpoint for injecting SLAS private client secret.
         const baseUrl = config.proxy.split(MOBIFY_PATH)[0]
         const privateClientEndpoint = `${baseUrl}${SLAS_PRIVATE_PROXY_PATH}`
-
+        const keysExcludedFromSuffixing = ['dw_dnt']
```

Comment:
```suggestion
        const DO_NOT_TRACK_KEY = ['dw_dnt']
```

@alexvuong is this what you mean?

Reply:
I was interpreting it to be something like:
```
const DO_NOT_TRACK_KEY = 'dw_dnt'
const keysExcludedFromSufixing = [DO_NOT_TRACK_KEY]
```

Reply:
That looks good to me @jeremy-jung1  👍

Reply:
Ended up not exactly looking like my example because the const has to be scoped to the Auth class. I made the private AuthDataKeys `this.dntCookieName` to be more specific because I didn't see another const with all caps in the auth index file. It made sense to type the `dw_dnt` as AuthDataKeys because `dw_dnt` is already in there and this way it types well with other function parameters